purchase_order_approved/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Purchase Order Approved
+=======================
+
+This module extends the functionality of purchases adding a new state
+*Approved* in purchase orders. Additionally add the possibility to set back to
+draft a purchase order in all the states previous to *Purchase Order*.
+
+In this new *Approved* state:
+
+* You cannot modify the purchase order.
```

Comment:
It is a little bit contradictory as is with the following sentence.
Why not: "In this state, you cannot modify the purchase order."